modules/user_manual/pages/files/manual_file_locking.adoc

```diff
@@ -11,6 +11,8 @@ All storages are supported as locking takes place on the WebDAV level. The locks
 
 NOTE: The _user-facing_ components in the web interface are disabled by default and must be enabled by the ownCloud administrator, because this feature allows users to lock other users files *exclusively*. Lock releasing can only be done by the locking user, a user who is a member of a lock breaking group (if defined by the ownCloud administrator) or until the lock expires.
 
+NOTE: Admin users control _Manual File Locking_ in _Settings -> _Admin_ -> _General_.
```

Comment:
```suggestion
NOTE: The settings for _Manual File Locking_ can be accessed at menu:Settings[Admin > General].
```

To be clarified, because if you look into the documentation, the menu already described is in `Additional` and not `General`. And as far I can remember, this got changed once to the version which is already in the documentation...
But if you look, see below, it is already documented...??
`Administrators can enable Manual File Locking for users either via the web interface or by executing an occ command:`
![image](https://user-images.githubusercontent.com/3321281/151561771-61e0e09c-4053-4062-8cf7-ca56e70db126.png)